featherpad/data/translations/featherpad_he.ts

```diff
@@ -1258,6 +1258,8 @@
         <translation>
             <numerusform>החלפה אחת</numerusform>
             <numerusform>%Ln החלפות</numerusform>
+            <numerusform>%Ln החלפות</numerusform>
+            <numerusform>%Ln החלפות</numerusform>
```

Comment:
@omeritzics
Thanks a lot for your translations!

I'm merging this change but please don't add the above two lines in future! Qt will remove them automatically, and we'd better to stick with what Qt considers correct.